aether-ui/aether/ui/assets/apps/pipeline/PipelineList.jsx

```diff
@@ -114,66 +123,120 @@ class PipelineList extends Component {
         } header={this.state.errorHeader} show={this.state.showError}>
           {this.state.errorMessage}
         </Modal>
-        <div
-          onClick={() => { this.onSelectPipeline(pipeline) }}>
-          { pipeline.is_read_only &&
+        <div className={`preview-input ${pipeline.isInputReadOnly ? 'pipeline-readonly' : ''}`} onClick={this.onSelectPipeline.bind(this, pipeline)}>
+
```

Comment:
Note: search for a javascript linter that complains with looooooooooooooooooooooooooog lines :unamused: